services/ui/serverless.yml

```diff
@@ -308,17 +314,6 @@ resources:
     DeliveryRole:
       Type: AWS::IAM::Role
```

Comment:
This doesn't need the IAM Path and Permission Boundary?  I'm confused how the feature branch deployed.